### ISSUE TITLE
Data Issue:  Validate new cases for Massachusetts Nov 20-22

### ISSUE THREAD

**Issue by MehraSakshi** [nytimes/covid-19-data#648]
We have searched for the new cases for Massachusetts, showing 7141 for 22nd November and 0 for 20th and 21st November, 2021.

Same data can be seen on NYT (https://www.nytimes.com/interactive/2021/us/massachusetts-covid-cases.html) and CDC(https://covid.cdc.gov/covid-data-tracker/#trends_dailycases).

But the data is different on Gov Website (https://www.mass.gov/info-details/covid-19-response-reporting#covid-19-interactive-data-dashboard-) and Worldometer (https://www.worldometers.info/coronavirus/usa/massachusetts/).

Github source: https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-states.csv 

Can you please check and confirm if the stats for Massachusetts are correct on github?

**Comment by tiffehr** [comment]
Our source for Massachusetts is [their Tableau dashboard](https://datavisualization.dph.mass.gov/views/MADPHCOVID-19Dashboard/COVID-19CasesOverTime?%3Aembed=y&%3AshowVizHome=n).  We check the page every hour, if not more.

**Today's numbers, for reference**
![image](https://user-images.githubusercontent.com/60173/146036671-84acc64e-e8c2-4d3e-b7a8-a37adf53493b.png)

Our figures are always cumulative totals that the state reports for that given day, and if the numbers don't change day over day, we show zero.  Our raw data collection reported: 
* `829,577` confirmed cases and `890,733` probable cases on 11/19/2021 in the evening ET
* `836,378` confirmed `897,874` probable cases on 11/22/2021 in the evening ET

We saw no new increases between those dates, which means the state did not update their state-level figure.  They may have _retroactively_ reported new confirmed/probable cases on those dates in breakout charts, but did not do so on the day we collected figures.  Our [README](https://github.com/nytimes/covid-19-data#methodology-and-definitions) has a lot of information, but includes that "All cases and deaths are counted on the date they are first announced."  But if the state/county doesn't announce a change, we don't retroactively try to find their figures unless it's a serious lull.  This is why our own display focuses on rolling averages, to smooth out days with that provided.
